src/data/types/studentType.js

```diff
@@ -14,24 +14,24 @@ import {
   GraphQLNonNull as NonNull,
 } from 'graphql';
 
-const HostelType = new ObjectType({
-  name: 'Hostel',
-  fields: {
-    hostelName: { type: StringType },
-    hostelRoom: { type: StringType },
-  },
-});
+// const hostelType = new ObjectType({
```

Comment:
Can I know why was this change made?

Reply:
@sharanry @SebastinSanty

Reply:
Maybe because we have changed the hostel table.. now its `hostelPs`. Basically its connected to two different tables `hostel` and `ps` where hostel or ps details are stored respectively. But I am not sure, I didnt make this change.